lib/capybara/poltergeist/client/compiled/web_page.js

```diff
@@ -107,16 +109,27 @@ Poltergeist.WebPage = (function() {
     });
   };
 
-  WebPage.prototype.onResourceRequestedNative = function(request) {
-    this.lastRequestId = request.id;
-    if (request.url === this.redirectURL) {
-      this.redirectURL = null;
-      this.requestId = request.id;
+  WebPage.prototype.onResourceRequestedNative = function(request, net) {
+    var abort, _ref2;
+    abort = this.urlBlacklist.some(function(blacklisted_url) {
+      return request.url.indexOf(blacklisted_url) !== -1;
+    });
+    if (abort) {
+      if (_ref2 = request.url, __indexOf.call(this._blockedUrls, _ref2) < 0) {
+        this._blockedUrls.push(request.url);
+      }
+      return net.abort();
```

Comment:
@route, it seems that my problems were local configuration. I'm on that old and known situation: "I can't repeat it!!!" :unamused:

What do you think? I remember that the situation was in this `net` param. It looked like it was undefined.

Do you know a situation that it can be undefined? Do you think that test its value before using it can be a good solution, just for be sure?